SharpXMPP.Shared/XmppTcpConnection.cs

```diff
@@ -121,34 +144,128 @@ public override void SessionLoop()
             }
         }
 
-        public override async void Connect()
+        // For backward compatibility
+        public override async void Connect() => await ConnectAsync();
```

Comment:
`async void` is a terrible solution and won't keep any "compatibility" for us. Let's remove that, please.

I don't think that keeping backward compatibility is that big issue for `v0.0.1` currently, but we need to discuss that with @vitalyster.

Reply:
I'm agree to remove